### DISCUSSION TITLE
What model can be used for realtime inference on edge devices?

### DISCUSSION THREAD

**Discussion by hardikdava** [open-edge-platform/anomalib#692] [Q&A]
I want to know the possiblity of doing realtime inference on edge devices like jetson devices or myriad-x.

- what model is fast enough to run on edge devices?
- is it possible to replace feature extractor from resnet to some small feature extractor?

It would be great if i get some interesting insights. Thank you.

**Comment by samet-akcay** [comment]
@ashwinvaidya17, are you gonna be able to provide some insights here?

**Reply by ashwinvaidya17** [reply]
Sure. I can write a script to compute FPS on NCS2.

**Reply by hardikdava** [reply]
Thank you, @samet-akcay and @ashwinvaidya17
